src/main/java/io/vertx/core/impl/IsolatingClassLoader.java

```diff
@@ -63,6 +66,59 @@
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public URL getResource(String name) {
+
+    // First check this classloader
+    URL url = findResource(name);
+
+    // Then try the parent if not found
+    if (url == null) {
+      url = super.getResource(name);
+    }
+
+    return url;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Enumeration<URL> getResources(String name) throws IOException {
+    @SuppressWarnings("unchecked")
+    Enumeration<URL>[] enums = (Enumeration<URL>[]) new Enumeration<?>[2];
```

Comment:
Wouldn't it be simpler to just put them in an ArrayList then return Collections.enumeration(list) ?